src/analysis/retail/shaman/restoration/modules/talents/EarthenWallTotem.tsx

```diff
@@ -250,6 +255,45 @@ class EarthenWallTotem extends Analyzer {
       />
     );
   }
+
+  /** Guide subsection describing the proper usage of Earthen Wall Totem */
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink id={TALENTS.EARTHEN_WALL_TOTEM_TALENT.id} />
+        </b>{' '}
+        does a lot of healing, particularly on stacked fights or whenever a fight has several damage
+        moments happen in quick succession. This totem does a ton of healing per cast and is an
+        excellent choice for <SpellLink id={TALENTS.TOTEMIC_RECALL_TALENT} />
+      </p>
+    );
```

Comment:
This isn't really saying a lot imo, could probably be reworded to describe how it's used as a mini healing cd for damage events etc